### ISSUE TITLE
Sanity Check Markup

### ISSUE THREAD

**Issue by duncdrum** [hcts-hra/ecpo-data#152]
Since validation was ignored for quite some time, there are several instances of either inconsistent or faulty markup. 

see #151 which brings back basic sanity to the markup structure. 

We need to ensure that markup is not only valid, but also correct and consistent when encoding textual features. 
That means:
- fix tag abuse: if we encode a textual features it needs to be explicit, not re-using existing tags for a different purpose. 
   - there is some confusion about the use of `div` and `p` in particular  see #150
- fix inconsistentcies: Once a feature is encoded on a single page, it needs to be consistently encoded in all pages. Similarly components should be consistently encoded across different pages. So identical header components, use identical markup across pages. See #79 #146
- Fix erroneous markup. 
   - `lb` has been inserted in places where no line breaks occur, these errors need to be fixed
- Update the docs/wiki, there are invalid examples in the wiki these need to be fixed
- there are only limited number of item types, replace existing examples with valid templates in the wiki for how to deal with, e.g. an article with inline images see #128, and article containing a graphic, the incipit of an article, etc. consistently. 
- fix superfluous markup: the following examples seem to simply introduce markup for the sake of it. These should be fixed, to make the file leaner. 

### Anti-patterns

one `p` per character
```xml
<div>
  <p>字<lb/></p>
  <p>字<lb/></p>
  <p>字<lb/></p>
…
</div>
```

This is simply
```xml
<p>
字<lb/>
字<lb/>
字<lb/>
…
</p>
```

interleaving div and p (without type or any other info)

```xml
<div type="article">
  <div>
    <p>字…</p>
  <div>
  <p>字…</p>
</div>
```

```xml
<div type="article">
  <p>字…</p>
  <p>字…</p>
</div>
```

incomprehensible tree

```xml
<div type="image">
  <div type="article">
  </div>
  <p>字…</p>
</div>
```

is there really such a thing as an article embedded in an image, but without any text, and some freefloating text that is both an image that is however not part of the article?

**Comment by xiejia1995** [comment]
for double-check:
https://kjc-sv034.kjc.uni-heidelberg.de/ecpo/admin/article/article_edit.php?magazin_id=1&article_id=650&page=7&issue_id=24

**Comment by duncdrum** [comment]
closed by #155
